src/vos/vos_internal.h

```diff
@@ -887,8 +887,8 @@ struct vos_iter_ops {
 	int	(*iop_copy)(struct vos_iterator *iter,
 			    vos_iter_entry_t *it_entry, d_iov_t *iov_out);
 	/** Delete the record that the cursor points to */
-	int	(*iop_delete)(struct vos_iterator *iter,
-			      void *args);
+	int	(*iop_process)(struct vos_iterator *iter, vos_iter_proc_op op,
```

Comment:
Compared with PR#5501, this seems more general solution. Currently, we set a DTX entry as 'corrupted' via vos_dtx_abort_one(). I prefer to unify them via the same API if possible. This patch offers some way for that.